e3sm_diags/driver/utils/dataset.py

```diff
@@ -281,7 +281,10 @@ def get_ref_filename_climo(self, season):
         path = self.parameters.reference_data_path
         data_name = self.parameters.ref_name
 
-        if hasattr(self.parameters, "ref_file"):
+        if (
```

Comment:
Add change to reconcile with default "ref_file" setting.

Reply:
Thanks!